lib/node_modules/@stdlib/stats/base/dsmeanwd/docs/repl.txt

```diff
@@ -87,9 +84,8 @@
 
     // Using offset parameter:
     > var x = new {{alias:@stdlib/array/float32}}( [ 1.0, -2.0, 3.0, 2.0, 5.0, -1.0 ] );
-    > var N = {{alias:@stdlib/math/base/special/floor}}( x.length / 2 );
-    > {{alias}}.ndarray( N, x, 2, 1 )
-    ~-0.3333
+    > {{alias}}.ndarray( 4, x, 2, 1 )
+    NaN
```

Comment:
This result does not look correct. The reason is the first argument should be `2`, not `4`.